profiles/chandramohan0.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/77689644/201625550-6ad7c9d2-a17e-462b-85c0-abf0cde09ddb.jpg",
+  "issueId": 1529,
+  "name": "Chandra Mohan",
+  "username": "chandramohan0"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```